Add email sending out kit disclaimer form early (#30)
srobo/team-emails: SR2020/2019-10-21-kit-distribution-and-kickstart-details.md

---
to: sr2020-teams
subject: Kit Distribution & Kickstart details
attachments:
  - https://drive.google.com/a/studentrobotics.org/file/d/1aCHnZ6o8DkLzHef1C2lz_iIkZkOjYP6P/view?usp=sharing
---

Hi all,

> Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details of shipping your kits.

Reviewer: ```suggestion
Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits will be ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details of shipping your kits.
```

> Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details of shipping your kits.

Reviewer: ```suggestion
Kickstart is less than a week away, and we're putting the finishing touches to kickstart. If you're attending a kickstart event, your kits are ready and waiting at the venue. For those who can't attend kickstart, we'll contact you shortly to confirm details for shipping your kits.
```

Reviewer: Actually on second thought, move the last sentence to the end of the next paragraph. (Makes it clearer that we want normal attendees to sign the form too, otherwise I think it could be misread if being scanned)

Before we can issue a kit, you must sign a kit disclaimer form. Historically these have been signed at the event, but we're sending them over early to speed up the issuing process on the day. If you're not attending Kickstart, return your form promptly to ensure you receive your kit as soon afterwards as possible.

Attached to this email is the disclaimer form. Please read it, print it, sign it, and [send it back to us](mailto:teams@studentrobotics.org). Alternatively, feel free to sign it and bring it with you. We will of course have some spares at the event to sign on the day.

> Please only attend the kickstart we're expecting you at: https://docs.google.com/spreadsheets/d/1plZqwcO0WIdoWhiUNdGxy-6uLSoH8vIsFdKzwHXtf0c/edit?usp=sharing. If you can no longer attend, please let us know.

Reviewer: ```suggestion
Please only attend the kickstart we're expecting you at: https://docs.google.com/spreadsheets/d/1plZqwcO0WIdoWhiUNdGxy-6uLSoH8vIsFdKzwHXtf0c/edit?usp=sharing. If you can no longer attend, please let us know.

If you're going to the Cambridge or London kickstarts, don't forget to remind your team to bring laptops, we won't have desktops available for you to use.
```

## Laptops

> If you're attending a kickstart, please be sure to remind competitors to bring a laptop, we won't have any desktops available for you to use. 1 per team is sufficient.

Reviewer: not sure '1 per team is sufficient' is correct, especially in the case of Hills Road. Maybe best to just say 'bring at least one'?

Author: good point!
